app/views/users/new.js.erb

```diff
@@ -1,7 +1,22 @@
-
-console.log('registration')
-
 var modalContent = '<%= escape_javascript(render partial: 'new') %>'
 App.mainContent.style.visibility = "hidden";
 App.tempContent.innerHTML = modalContent
-closeModalListener('cancel_modal')
\ No newline at end of file
+closeModalListener('cancel_modal')
+
+
+var form = document.getElementById('create_user')
+
+form.addEventListener('ajax:error', (event) => {
+    let errorMessageDisplayElement = document.getElementById('error_messages')
+    let message = event.detail[0].message
+    errorMessageDisplayElement.innerHTML = message
+})
+
+form.addEventListener('ajax:success', (event) => {
+    let errorMessageDisplayElement = document.getElementById('messages')
+    let message = event.detail[0].message
+    let model = document.getElementById('modal')
+    errorMessageDisplayElement.innerHTML = message
```

Comment:
Need to change this name from errorMessageDisplayElement to messageDisplayElement on line 16 and 19.

Reply:
Updated. We decided to cancel the (ajax) event listener for displaying success messages, as we are already doing it through the controller.